Update DCP_change_log.md (#3195)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

> [!NOTE]
> Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).

#### 29 May 2024 - Enhancement - Popup buttons [ID_39010]

> The buttons appearance in popups have been updated.

Reviewer: ```suggestion
Buttons in the popups have been updated to the new style.
```

> #### 29 May 2024 - Enhancement - Popup buttons [ID_39010]
> 
> Buttons in the popups have been updated to the new style.

Reviewer 2: See https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/pop-up
```suggestion
#### 29 May 2024 - Enhancement - New style for buttons in pop-up windows [ID_39010]

Buttons in pop-up windows have been updated to the new style.
```

#### 29 May 2024 - Fix - Admin - email validation [ID_39791]

> The email validation has been corrected.

Reviewer: Give some more explanation on what is supported now

> #### 29 May 2024 - Fix - Admin - email validation [ID_39791]
> 
> The email validation has been updated to support all domain extensions.

Reviewer 2: As this is phrased as an enhancement, I'd mention this as an enhancement rather than a fix:

```suggestion
#### 29 May 2024 - Enhancement - Admin - email validation [ID_39791]

Email validation in the Admin app has been updated to support all domain extensions.
```

If you do want to mention it as a fix, you'll instead need to adjust the description so it describes what went wrong from the perspective of the user.

Author: It still needs to be described as a fix. I changes the sentence.

> #### 29 May 2024 - Enhancement - Admin - DMS overview failover pair offline status [ID_39694]
> 
> The DMS overview page will now correctly show the failover pair agent as offline when this is the case.

Reviewer 2: "Failover" should always be capitalized if it's a reference to the DataMiner Failover feature. Similarly, "Agent" should always be capitalized if it's a reference to a DataMiner Agent.

```suggestion
#### 29 May 2024 - Enhancement - Admin - DMS overview Failover pair offline status [ID_39694]

The DMS overview page will now correctly show a Failover Agent as offline when appropriate.
```
